pkg/csi/recover/recover.go

```diff
@@ -53,26 +66,92 @@ type containerStat struct {
 	startAt       metav1.Time
 }
 
-func NewFuseRecoder(kubeClient client.Client, kubeletClient *kubelet.KubeletClient, recorder record.EventRecorder) *FuseRecover {
+func initializeKubeletClient() (*kubelet.KubeletClient, error) {
+	// get CSI sa token
+	tokenByte, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/token")
```

Comment:
If this is a fixed value, it is recommended to put it in a uniform location

Reply:
Fixed.